src/pagination/pagination.js

```diff
@@ -43,11 +42,12 @@ angular.module('ui.bootstrap.pagination', [])
   };
 
   this.render = function() {
-    $scope.page = parseInt(ngModelCtrl.$viewValue, 10) || 1;
+      $scope.page = parseInt(ngModelCtrl.$viewValue, 10) || 1;
   };
 
   $scope.selectPage = function(page, evt) {
-    if ( $scope.page !== page && page > 0 && page <= $scope.totalPages) {
+    var disabled_click = $scope.paginationDisabled && evt !== undefined;
```

Comment:
Use a camelcased variable - `disabled_click` is bad JS code style.

In addition, `$scope.paginationDisabled && evt` is a simpler to read check since checking for definedness is all the statement seems to be doing.